frontend/fingertips-frontend/app/chart/page.tsx

```diff
@@ -30,12 +30,31 @@ export default async function ChartPage(
   await connection();
 
   const indicatorApi = ApiClientFactory.getIndicatorsApiClient();
+  const areaApi = ApiClientFactory.getAreasApiClient();
+
+  let parentAreaCode: string | undefined;
+
+  if (indicatorsSelected.length === 1 && areaCodes.length <= 2) {
+    try {
+      const areaData = await areaApi.getArea({ areaCode: areaCodes[0] }); // DHSCFT-256 assumes one common parent
+      if (areaData?.parent) {
+        parentAreaCode = areaData.parent.code;
```

Comment:
This could be simplified with something like this:  parentAreaCode = areaData?.parent?.code;  

This will do the null checking for you and parentAreaCode will be set to null, even if areaData is null.